test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -581,3 +581,27 @@
          (is (= 204 (:status response)))
          (is (not (fs/exists? saved-csr)))))
      (fs/delete csr-file))))
+
+(deftest ca-expirations-endpoint-test
+  (testing "returns expiration dates for all CA certs and CRLs"
+    (bootstrap/with-puppetserver-running-with-mock-jrubies
+     "JRuby mocking is safe here because all of the requests are to the CA
+     endpoints, which are implemented in Clojure."
+     app
+     {:jruby-puppet
+      {:gem-path [(ks/absolute-path jruby-testutils/gem-path)]}
+      :webserver
+      {:ssl-cert (str bootstrap/master-conf-dir "/ssl/certs/localhost.pem")
+       :ssl-key (str bootstrap/master-conf-dir "/ssl/private_keys/localhost.pem")
+       :ssl-ca-cert (str bootstrap/master-conf-dir "/ssl/ca/ca_crt.pem")
+       :ssl-crl-path (str bootstrap/master-conf-dir "/ssl/crl.pem")}}
+     (let [options {:ssl-cert (str bootstrap/master-conf-dir "/ssl/ca/ca_crt.pem")
+                    :ssl-key (str bootstrap/master-conf-dir "/ssl/ca/ca_key.pem")
+                    :ssl-ca-cert (str bootstrap/master-conf-dir "/ssl/ca/ca_crt.pem")
+                    :as :text}
+           response (http-client/get "https://localhost:8140/puppet-ca/v1/ca-expirations"
+                                     (merge options {:headers {"Accept" "application/json"}}))]
```

Comment:
why `merge` instead of just including `:headers` when you define `options`?

Reply:
Copied from another test. Probably don't need the `let` at all here since there's no repetition.